src/js/components/TextInput/textinput.stories.js

```diff
@@ -28,6 +27,26 @@ class SimpleTextInput extends Component {
   }
 }
 
+const PasswordInput = ({ value, ...rest }) => {
+  const [inputValue, setValue] = useState(value);
+  const [reveal, setReveal] = useState(false);
+  return (
+    <Box width="medium" margin="large" round="small" border>
+      <TextInput
+        plain
+        type={reveal ? 'text' : 'password'}
+        value={inputValue}
+        onChange={event => setValue(event.target.value)}
+        {...rest}
+      />
+      <Button
```

Comment:
The button is taking the full space and the focus looks awkward 
![screen shot 2019-03-01 at 6 03 08 pm](https://user-images.githubusercontent.com/6320236/53674624-6c14e300-3c4c-11e9-9e41-d400d5b1312c.png)

Not a must, bet will be better to have it fixed.

Reply:
oups forgot row and align.